packages/app-project/stores/User/Recents/Recents.spec.js

```diff
@@ -34,6 +34,11 @@ describe('Stores > Recents', function () {
     rootStore = initStore(true, { project })
     recentsStore = rootStore.user.recents
     sinon.stub(rootStore.client.panoptes, 'get').callsFake(() => Promise.resolve(mockResponse))
+    sinon.stub(rootStore.client.collections, 'get').callsFake(() => Promise.resolve({
```

Comment:
I don’t think so, because `rootStore` is discarded once the tests have run. 

These tests could be rewritten to mock the Panoptes API with `nock`, though.